src/router.js

```diff
@@ -12,15 +12,9 @@ import HostingEarnings from 'pages/HostingEarnings'
 import MyProfile from 'pages/MyProfile'
 import StyleDemo from 'pages/StyleDemo'
 import Login from 'pages/Login'
-import ScreenWidthContext from 'contexts/screenWidth'
-import styles from './PrimaryLayout.module.css' // eslint-disable-line no-unused-vars
-import 'global-styles/colors.css'
-import 'global-styles/index.css'
-
-export function PrimaryLayout () {
-  const isWide = useContext(ScreenWidthContext)
 
-  return <div styleName={cx('styles.primary-layout', { 'styles.wide': isWide }, { 'styles.narrow': !isWide })}>
+export default function HPAdminRouter () {
```

Comment:
Router file - pattern that I have mentioned back in Berlin discussions.
Single place to define our URL structure, which doesn't care about appearance.
IMHO quite readable and helpful practice ;)